pyproject.toml

```diff
@@ -26,7 +26,10 @@ classifiers = [
 ]
 description-file = "README.md"
 requires-python = ">=3.6"
-
```

Comment:
Maybe you should add here:
[build-system]

Reply:
We already have it at the beginning of the file
![image](https://user-images.githubusercontent.com/34128882/124927271-1e23fd00-e007-11eb-8ac3-33cdf84d7219.png)

Reply:
So those:
```
[
    "black>=20.8b1,<21.0",
    "isort[requirements_deprecated_finder,pipfile_deprecated_finder]>=5.8.0,<6.0.0",
]
```
Should be under the same key above: 
`requires`